homeworks/HW1/src/app-router.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import {Route, Link, Switch} from "react-router-dom"
+import {Router} from "react-router";
+import createBrowserHistory from 'history/createBrowserHistory';
+import {CoursesViewer} from './features/courses/viewer';
+import {Login} from './features'
+import {CoursesCreator} from './features/courses/creator';
+import {ErrorPageNotFound} from './features/page-not-found';
+import {AuthorizationChecker} from './features/user/login/components/auth-checker';
+
+// ** THIS VARIANT NEEDS TO BE REVIEWED **
+// async.js is applied in common/components/async
+//
+// const Login = asyncComponent(() =>
+//     import('./features/user/login').then(module => module.default)
```

Comment:
That is not worked because you import `default` from the module, but in every files,you are export not default. So if you will get from module your component then should be work.

Reply:
thank you, that's it
it works :)